openfecli/parameters/molecules.py

```diff
@@ -69,3 +69,10 @@ def molecule_getter(user_input, context):
     ),
     getter=molecule_getter,
 )
+
+COFACTORS = Option(
```

Comment:
Two things:

1. This behaviour is difference than "MOL_DIR", do we expect folks to sometimes pass a directory of molecule files for spectator molecules?
2. This might be a bit late, but the fact that we use -c and -M and -p is just annoying.

Reply:
`-m` is already taken by a single-molecule version that is used elsewhere. Maybe capital letters when they're expected to contain multiple objects

Reply:
was `-l` not available? (l for ligand)